plugins/factory.go

```diff
@@ -167,6 +167,31 @@ var strategies = map[string]StrategyContainer{
 			return s, nil
 		},
 	},
+	"buy_twap": {
+		SortOrder:   7,
+		Description: "Creates buy offers by distributing orders over time for a given day using a twap metric",
+		NeedsConfig: true,
+		Complexity:  "Intermediate",
+		makeFn: func(strategyFactoryData strategyFactoryData) (api.Strategy, error) {
+			var cfg sellTwapConfig
```

Comment:
can you add a comment here saying that we are just reusing the `sellTwapConfig` struct in the code because it's exactly the same as what we need for the buyTwap strategy?